src/WebJobs.Script/OutOfProc/Http/MessageExtensions/HttpMessageExtensionUtilities.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Dynamic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.OopHttp
```

Comment:
Why the different namespace here?

Reply:
fixed namespace! Same here the purpose of this converter is to convert from HttpResponse message to http output binding.